databases/week3/meal_sharing.sql

```diff
@@ -0,0 +1,290 @@
+CREATE DATABASE meals;
+SET
+  NAMES utf8mb4;
+USE meals;
+CREATE TABLE `meal`
+(
+  `id` INT(10) UNSIGNED NOT NULL PRIMARY KEY AUTO_INCREMENT,
+  `title` VARCHAR(255) NOT NULL ,
+  `description` TEXT NULL DEFAULT NULL,
+  `location` VARCHAR(255),
+  `when` DATETIME NOT NULL,
+  `max_reservations` INT(50) UNSIGNED,
+  `price` DECIMAL UNSIGNED NOT NULL,
+  `created_date` DATETIME NOT NULL DEFAULT NOW()
+)ENGINE = InnoDB DEFAULT CHARSET = utf8mb4 COLLATE = utf8mb4_unicode_ci;
+CREATE TABLE `reservation`
+(
+  `id` INT(10) UNSIGNED NOT NULL PRIMARY KEY AUTO_INCREMENT ,
+  `no_of_guests` INT(50) UNSIGNED NOT NULL ,
+  `meal_id` INT(10) UNSIGNED NOT NULL,
+  `created_date` DATETIME NOT NULL DEFAULT NOW(),
+  `contact_phonenumber` VARCHAR(255) NOT NULL,
+  `contact_name` VARCHAR(255) NOT NULL,
+  `contact_email` VARCHAR(255) NOT NULL,
+   CONSTRAINT `fk_reservation_meal` FOREIGN KEY (`meal_id`) REFERENCES `meal` (`id`) ON DELETE CASCADE ON UPDATE CASCADE
+)ENGINE = InnoDB DEFAULT CHARSET = utf8mb4 COLLATE = utf8mb4_unicode_ci;
+CREATE TABLE `review`
+(
+  `id` INT(10) UNSIGNED NOT NULL PRIMARY KEY AUTO_INCREMENT,
+  `title` VARCHAR(255) NOT NULL,
+  `description` TEXT NULL DEFAULT NULL,
+  `meal_id` INT(10) UNSIGNED NOT NULL,
+  `stars` INT(10) NOT NULL,
+  `created_date` DATETIME NOT NULL DEFAULT NOW(),
+  CONSTRAINT `fk_review_meal` FOREIGN KEY (`meal_id`) REFERENCES `meal` (`id`) ON DELETE CASCADE ON UPDATE CASCADE
+)ENGINE = InnoDB DEFAULT CHARSET = utf8mb4 COLLATE = utf8mb4_unicode_ci;
+
+INSERT INTO
+  meal (
```

Comment:
All of these newlines decrease readability imho. Maybe have each entry be on a single line

Reply:
Thank you so much for the great comments. I have updated all the feedback.